src/webview/html/message.js

```diff
@@ -52,33 +77,45 @@ const messageReactionAsHtml = (
     allImageEmojiById[reaction.code]
       ? template`<img src="${allImageEmojiById[reaction.code].source_url}"/>`
       : codeToEmojiMap[reaction.code]
-  }&nbsp;${reaction.count}</span>`;
+  }&nbsp;${
+    // The web app puts this condition in get_vote_text in its reactions.js.
+    shouldShowNames
+      ? getDisplayFullNames(reaction.users, backgroundData, _).join(', ')
+      : reaction.count
+  }</span>`;
 };
 
 const messageReactionListAsHtml = (
   backgroundData: BackgroundData,
   reactions: $ReadOnlyArray<Reaction>,
+  _: GetText,
 ): string => {
-  const { ownUser } = backgroundData;
+  const { ownUser, displayEmojiReactionUsers } = backgroundData;
 
   if (reactions.length === 0) {
     return '';
   }
+
+  // See the web app's get_vote_text in its reactions.js.
```

Comment:
Hmm, I'm not finding a function by that name.

Reply:
Ah, I see, it's added by https://github.com/zulip/zulip/pull/21013 .

Reply:
Ah, yeah, I was looking at the `chat.zulip.org` branch when putting this together.